bpf/lib/conntrack.h

```diff
@@ -355,6 +367,14 @@ __ct_lookup(const void *map, struct __ctx_buff *ctx, const void *tuple,
 	}
 
 ct_new: __maybe_unused;
+#ifdef ENABLE_ACTIVE_CONNECTION_TRACKING
+	if (dir == CT_SERVICE) {
+		if (ct_state)
+			_lb_act_conn_open(ct_state->rev_nat_index, ct_state->backend_id);
+		else if (entry)
```

Comment:
I believe this would happen too early in the code flow now. For a new connection, we haven't selected a backend yet.